hashicorp/waypoint
# ui: ember 3.28

## Why the change?

Ember 3.28 is now an LTS release. We don’t necessarily have to upgrade now, but I needed a distraction from something more challenging and thought it’d be useful to know the following:

- Is anything stopping us upgrading?
- How many deprecations do we have to attend to?

Note that this re-uses work from #2738 which will unfortunately have the same issue of confusing ember-asset-size-action.

## What’s the plan?

- [x] Upgrade everything that needs upgraded ([official guide here](https://blog.emberjs.com/ember-4-0-released#toc_how-to-upgrade-your-project-to-ember-4))
- [x] Decide what to do about ember-asset-sizes

## How do I test it?

1. `git checkout ui/ember-3.28`
2. [Boot the dev server](https://github.com/hashicorp/waypoint/blob/main/ui/README.md#running--development)
3. Smoke test
4. Verify nothing obvious is broken
